README.md

```diff
@@ -28,15 +28,3 @@ For some tasks, the CASL repository has several dependencies on external reposit
 
 > **Note:** The dependencies are managed using `ansible-galaxy` and the specific instructions will call this out when there is a need to use galaxy to pull in the correct dependencies.
 
-Below is a list of versions of `openshift-ansible` the CASL team has leveraged and found to be stable/working. Note that other versions may work as well. Feel free to submit PRs for this file with updated versions as they are found to be functional.
```

Comment:
@oybed where are we moving this to? I only see a delete.

Reply:
@etsauer this is now being handled with `ansible-galaxy`, so we don't track anymore ... just use the latest `release-3.x` version:
https://github.com/oybed/casl-ansible/blob/v37/casl-requirements.yml#L12-L14

Reply:
It also simplifies the documentation as the users don't have to manually set up `openshift-ansible` anymore